.github/workflows/check-translation-keys.yml

```
@@ -0,0 +1,24 @@
+
+jobs:
+  build:
+    name: Check if translation keys are consistent
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Check if translation keys match
+        run: >
```

actionlint

shellcheck reported issue in this script: SC2215:warning:2:3: This flag is used as a command name. Bad line break or missing [ .. ]?

(shellcheck)